sig/scheduling/README.md

```diff
@@ -0,0 +1,29 @@
+# Scheduling Special Interest Group(Scheduling SIG)
+
+Scheduling SIG covers the scheduling system in TiDB cluster. Our work includes
```

Comment:
Should "TiDB cluster" be changed into "the TiKV client"?

Reply:
No.

Reply:
We should definitely repurpose this SIG - I think firstly it's the scheduling system for PD. For TiDB, we can say that TiDB clusters also depend on PD as the scheduling center.

Reply:
How about "Scheduling SIG covers the scheduling system for TiKV"? Do we need to mention TiDB here?

Reply:
Sounds good to me. We don't need to mention TiDB here. But we may add a section like "Related SIGs or Related projects" so that we can list the work or projects that are closely related to PD.